javascript-and-json/main.js

```diff
@@ -0,0 +1,28 @@
+var books = [
+  {
+    isbn: 9780553204742,
+    title: 'Where the Red Fern Grows',
+    author: 'Rawls, Wilson'
+  },
+  {
+    isbn: 9780812550702,
+    title: 'Enders Game',
+    author: 'Card, Orson Scott'
+  },
+  {
+    isbn: 9780749709303,
+    title: 'Little House on the Prarie',
+    author: 'Wilder, Laura Ingalls'
+  }
+];
+
+console.log('type of: ', typeof books);
+
+console.log(JSON.stringify(books));
+console.log('typeof: ', typeof JSON.stringify(books));
+
+var barbara = JSON.stringify({ 'Number id': 1, 'String name': 'Barbara' });
```

Comment:
Did you see my post in the class channel? I want you to write the actual string by hand here, not stringify an object you created.

Reply:
Oops, just saw it now!! Sorrryyyyyy :'(